### PR TITLE
add arch linux install

### PR DESCRIPTION
**Pull request by soloturn**
add arch linux install to README, to avoid people going through the lengthy install process there.

### CODE REVIEW

**Review by rexhent — CHANGES_REQUESTED** [review]
You accidentally doubled the link in line 121

**Comment by soloturn** [comment]
thanks for the review, @rexhent ! i guess i divorce [helix](https://github.com/helix-editor/helix) editor on xfce terminal, as it double pastes *grrrr*

**Comment by soloturn** [comment]
would you mind merging this or something is missing, @rexhent @Drakulix ?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # COSMIC Desktop
 
-Currently an incomplete **pre-alpha**.
+Currently an incomplete **pre-alpha**. Standard Installation is available for Pop!_OS, Arch Linux and its decendents, like Manjaro.
```
Comment (Drakulix) [inline README.md:3]: I don't mind having specific instructions for some distributions, but in general the systemd-sysext method works on a lot of distributions for testing. Which is all this should be used for right now. So please drop or adjust this line.

```suggestion
Currently an incomplete **pre-alpha**. Testing instructions below for various distributions.
```

File: README.md [hunk 2]
```diff
@@ -23,7 +24,7 @@ Currently an incomplete **pre-alpha**.
 * [libcosmic](https://github.com/pop-os/libcosmic)
 * [xdg-desktop-portal-cosmic](https://github.com/pop-os/xdg-desktop-portal-cosmic)
 
-## Setup
+## Setup without package
```
Comment (Drakulix) [inline README.md:27]:
```suggestion
## Setup on distributions without packaging of cosmic components
```

File: README.md [hunk 3]
```diff
@@ -128,7 +129,14 @@ sudo systemctl restart gdm
 
 After logging out, click on your user and there will be a sprocket at the bottom right. Change the setting to COSMIC. Proceed to log in.
 
-#### Configuring COSMIC DE
+## Installing on Arch Linux
+Installing via the preferred AUR helper is possible the usual way, e.g.:
+`paru -S cosmic-epoch-git`
+
+Then log out, click on your user, and a sprocket at the bottom right shows an additional entry alongside your desktop environments. Change to COSMIC and proceed with log in.
+For a more detailled discussion consider the [relevant section in the Arch wiki](https://wiki.archlinux.org/title/COSMIChttps://wiki.archlinux.org/title/COSMIC).
```
Comment (Drakulix) [inline README.md:137]: This seems like a copy&paste error?

```suggestion
For a more detailled discussion consider the [relevant section in the Arch wiki](https://wiki.archlinux.org/title/COSMIC).
```

**Comment by soloturn** [comment]
cool, first time i saw edit pull request and commit on web ...

**Review by Drakulix — APPROVED** [review]
Thanks!

**Pull request merged**